docs/pages/get-started/installation.mdx

```diff
@@ -18,6 +18,7 @@ To use Expo CLI, you need to have the following tools installed on your develope
 
 - [Node.js LTS release](https://nodejs.org/en/) - Only Node.js LTS releases (even-numbered) are recommended.
   As Node.js [officially states](https://nodejs.org/en/about/releases/), "Production applications should only use Active LTS or Maintenance LTS releases".
+  - Consider installing Node.js using [nvm](https://github.com/nvm-sh/nvm) or [nvm-windows](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows#install-nvm-windows-nodejs-and-npm) to easily install and switch between different versions
```

Comment:
Appreciate this addition and I understand the context and intention behind the change.

I don't think we should be opinionated about recommending a tool like a node version manager. There are other tools that people can use like `n` that some folks might prefer. Recommending one over the other, might sound like we are forcing our users to use one tool over the other. I'd say, let's keep things as it is on this line.

Reply:
I think it doesn't hurt to be a bit more opinionated. We're also opinionated on the workspace tooling we support, and the package managers (like, no official pnpm support, only yarn-workspaces support over turborepo, nx, etc.). There's a number of tools for node installation management (asdf, n, nvm, volta), and each have their own quirks at times. For newcomers, reducing the number of paths they can take makes our life significantly easier in the support department as well (we've especially seen that be the case on unsupported workspace configurations). TL;DR: opinionated isn't a bad thing sometimes.